### PR TITLE
[#182125818] Tomcat-Config Rolle nach Rheinwerk migrieren

### PR DESCRIPTION
**Pull request by eifelmicha**
- Zusammenführung der update_tomcat_config ASP rollen aus App/Rest Server + SalesClient nach Rheinwerk
- Skripte für Draining werden bei Bedarf mitkonfiguriert

### CODE REVIEW

**Comment by eifelmicha** [comment]
@dennis-kuypers 
requesting review.
requesting functional acceptance.
requesting merge approval.

File: templates/etc/ansible/playbooks/tomcat-drain.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    # We wait to avoid a situation where the server is already drained but consul never had time to
+    #  detect and propagate this information. This prevents shutting down an already drained server
+    #  at the same time as haproxy still allows new connections leading to errors.
+    #
+    # Wait time calculation
+    #   10s (consul health probe interval)
+    # + 30s (haproxy worst case dns update delay)
+    # +  2s (additional time for operations that we didn't think about)
+    # -----------------------------------------------------------------
+    #   10s total wait time
```
Comment (dennis-kuypers) [inline templates/etc/ansible/playbooks/tomcat-drain.yml.j2]: seems to be a typo, can you adjust the total wait time to say 42?

File: templates/etc/init/tomcat.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,67 @@
+description "Tomcat {{ _tomcat.role }}"
```
Comment (dennis-kuypers) [inline templates/etc/init/tomcat.conf.j2]: Can we drop this in favor of systemd?

**Review by dennis-kuypers — COMMENTED** [review]
see comments

File: templates/etc/ansible/playbooks/tomcat-drain.yml.j2 (hunk 1 above)
Comment (eifelmicha) [reply templates/etc/ansible/playbooks/tomcat-drain.yml.j2]: Copied from RestServer. Will fix.

File: templates/etc/init/tomcat.conf.j2 (hunk 2 above)
Comment (eifelmicha) [reply templates/etc/init/tomcat.conf.j2]: Yes

**Comment by eifelmicha** [comment]
@dennis-kuypers
requesting review.
requesting functional acceptance.
requesting merge approval.

**Comment by eifelmicha** [comment]
@dennis-kuypers
requesting review.
requesting functional acceptance.
requesting merge approval.

**Review by dennis-kuypers — APPROVED** [review]
Review: LGTM
Functionally accepted
Merge approved

**Pull request merged**